src/commands/remove-demo.ts

```diff
@@ -19,14 +19,17 @@ const MATCHING_GLOBS = [
 
 module.exports = {
   alias: ["rd", "remove-demos"],
-  description: "Remove demo code from generated boilerplate",
+  description:
+    "Remove demo code from generated boilerplate. Add --dry-run to see what would be removed.",
   run: async (toolbox: GluegunToolbox) => {
     const { parameters, patching, filesystem } = toolbox
 
     const CWD = process.cwd()
     const TARGET_DIR = parameters.first ?? CWD
+    const dryRun = parameters.options.dryRun ?? false
```

Comment:
Going to want to wrap this in a boolFlag, or else it will be `'true'`, `'false'`, or `undefined`